### PR TITLE
Update InternalListener.java

### PR DESCRIPTION
**Pull request by Ectabro**
To solve my issue  #266 this can be the solution ?

### CODE REVIEW

**Comment by portlek** [comment]
Please fix the build issue.

**Comment by Ectabro** [comment]
Okay now it should work sorry i take time to understand how to do that

File: src/main/java/org/maxgamer/quickshop/Event/ShopSuccessPurchaseEvent.java [hunk 1]
```diff
@@ -36,14 +37,15 @@
 
     @Getter
     @NotNull
-    private final Shop shop;
+    public final Shop shop;
```
Comment (portlek) [inline src/main/java/org/maxgamer/quickshop/Event/ShopSuccessPurchaseEvent.java]: In this case; the field must be private, and @Getter annotation will create getShop() method for it.

Comment (Ectabro) [reply src/main/java/org/maxgamer/quickshop/Event/ShopSuccessPurchaseEvent.java]: okay but i don't understand how to use this getShop() method in the InternalListener.java

**Comment by Ectabro** [comment]
Better ?

**Comment by portlek** [comment]
Install the Lombok plugin.

**Comment by Ectabro** [comment]
I tested my change and everything work perfectly

**Pull request merged**